datasets/sport/sport.json

```diff
@@ -0,0 +1,646 @@
+      "title": "Zwembad",
+      "type": "table",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["schema", "id"],
+        "display": "naam",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "id": {
+            "type": "integer",
+            "description": "Unieke aanduiding van de sportfaciliteit."
+          },
+          "geometry": {
+            "$ref": "https://geojson.org/schema/Geometry.json",
+            "description": "Puntgeometrie van de sportfaciliteit."
+          },
+          "stadsdeel": {
+            "type": "string",
+            "description": "Stadsdeel waaronder de sportfaciliteit valt."
+          },
+          "naam": {
+            "type": "string",
+            "description": "Naam van de sportfaciliteit."
+          },
+          "adres": {
+            "type": "string",
+            "description": "Adres van de sportfaciliteit."
+          },
+          "postcode": {
+            "type": "string",
+            "description": "Postcode van de sportfaciliteit."
+          },
+          "plaats": {
+            "type": "string",
+            "description": "Plaats van de sportfaciliteit."
+          },
+          "telefoonnummer": {
+            "type": "string",
+            "description": "Telefoonnummer van de sportfaciliteit."
+          },
+          "emailadres": {
+            "type": "string",
+            "description": "E-mailadres van de sportfaciliteit.",
+            "provenance": "e_mailadres"
+          },
+          "website": {
+            "type": "string",
+            "format": "uri",
+            "description": "Website van de sportfaciliteit."
+          },
+          "type": {
+            "type": "string",
+            "description": "Type sportfaciliteit."
+          },
+          "aantalHallen": {
+            "type": "string",
+            "description": "Aantal hallen (ruimtes) in de sportfaciliteit.",
+            "provenance": "aantal hallen"
+          },
+          "gebiedsgerichtwerkengebied": {
+            "type": "string",
+            "description": "Naam van gebiedsgerichtwerkengebied waaronder sportfaciliteit valt.",
+            "provenance": "naam gebiedsgerichtwerkengebied"
+          },
+          "vastgoedeigenaar": {
+            "type": "string",
+            "description": "Eigenaar van de sportfaciliteit.",
+            "provenance": "vastgoedeigendom"
+          },
+          "exploitant": {
+            "type": "string",
+            "description": "Exploitant van de sportfaciliteit.",
+            "provenance": "exploitatie"
+          },
+          "verhuuradministratie": {
+            "type": "string",
+            "description": "Verhuuradministratie instantie van de sportfaciliteit."
+          },
+          "xcoordinaat": {
```

Comment:
Ik zou voorstellen om de X en Y coordinaten hier weg te laten want die kunnen gewoon uit de geometrie gehaald worden via de Accept-Crs toch?

Reply:
Ja is prima, voor de verwerking heb ik ze wel nodig om een geometry object te maken, maar ik kan ze verwijderen na gebruik